facia/public/humans.txt

```diff
@@ -104,7 +104,6 @@ Engineer: Oliver Lloyd
 From: London, UK
 Location: London, UK
 
-
```

Comment:
Why no line here?

Reply:
Just to match the one-line paragraph spacing of the rest of the file cause I'm a clean freak

Reply:
I think this was my fault, thanks!